mapproxy/util/ext/serving.py

```diff
@@ -119,8 +123,24 @@ def run_wsgi(self):
 
         def write(data):
             assert headers_set, 'write() before start_response'
+            import pdb; pdb.set_trace()
```

Comment:
this should be removed

Reply:
ouch, yes of course, that slipped through and there is not CI test of the http server to catch this.
Thanks